src/Microsoft.DotNet.Interactive/CompositeKernel.cs

```diff
@@ -213,6 +213,22 @@ protected override void SetHandlingKernel(KernelCommand command, KernelInvocatio
                 throw new NoSuitableKernelException(command);
             }
 
+            switch (command)
+            {
+                case Cancel _:
+
+                    CancelInflightCommands();
+                    ClearPendingCommands();
```

Comment:
Is "pending" different from "deferred"?

Reply:
pending includes all things in the trampoline queue and the deferred

Reply:
So it might be clearer to say `ClearQueuedAndDeferredCommand` since both of those are terms that already exist in the code.